brd-ios/breadwallet.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 52;
```

Comment:
is it ok that this number decreased?

Reply:
I am on the latest version of Xcode. Should be fine